storage.template

```diff
@@ -52,33 +56,49 @@
     "VolumeSize": {
       "Description": "Size in GB of the volume",
       "Type": "String",
-      "Default": "16"
+      "Default": "50"
     },
     "ClusterSize": {
       "Description": "Members of the cluster, keep an odd number",
       "Type": "String",
-      "Default": "3"
+      "Default": "1"
```

Comment:
Curious why this would be set to 1 here as a default? I was originally thinking that most deployments would want minimum redundancy.